assets/dev/scss/frontend/_container.scss

```diff
@@ -18,10 +18,10 @@
 	--min-height: initial;
 	--height: auto;
 	--text-align: initial;
-	--margin-top: 0;
-	--margin-right: 0;
-	--margin-bottom: 0;
-	--margin-left: 0;
+	--margin-top: 0px;
+	--margin-right: 0px;
+	--margin-bottom: 0px;
+	--margin-left: 0px;
```

Comment:
Default values without a unit are giving problems inside `calc()` functions.

Reply:
Didn't @rami-elementor, in [his comment](https://elementor.atlassian.net/browse/ED-10309?focusedCommentId=123390), offered to leave the values without units?